src/components/Charts/LineChart.tsx

```diff
@@ -37,7 +36,7 @@ function LineChart<T>({
     <svg width={width} height={height}>
       <Group top={marginTop}>
         <LinePath
-          curve={curve ?? radius(0.25)}
+          curve={curve}
```

Comment:
curve should potentially be a required prop here now, or we can choose a new default

Reply:
ah, good catch. I made it required now